Bug: Fix errors in rpm/spec.in file

andreas-bok-sociomantic opened the pull request:
#Bogus date for v6.0.4 release

Bugfix: Use %{name} in lieu of %{header_install_path}
The variable `header_install_path` was uninitialized
and thus resulted in rpmbuild not being able to
find the header files.

daniel-zullo reviewed (approved):
The changes LGTM, I would only suggest to apply the changes in separate commits (if possible).

andreas-bok-sociomantic commented:
Everything is possible :)

andreas-bok-sociomantic committed:
Bugfix: Use %{name} in lieu of %{header_install_path}

The variable `header_install_path` was uninitialized
and thus resulted in rpmbuild not being able to
find the header files.

andreas-bok-sociomantic committed:
Bugfix: Bogus date for v6.0.4 release in changelog

daniel-zullo-sociomantic commented:
> Everything is possible :)

Then please solve the P versus NP problem (if it hasn't been solved yet)

andreas-bok-sociomantic commented:
Split changes into 2 commits. 

> Then please solve the P versus NP problem (if it hasn't been solved yet)

I will start by finding out how to make the perfect pesto

daniel-zullo-sociomantic commented:
Waiting for travis to rebase it.

andreas-bok-sociomantic commented:
TX :)
